### DISCUSSION TITLE
How to store settings in sub folder of FT installation directory?

### DISCUSSION THREAD

**Discussion by pstein** [FreeTubeApp/FreeTube#4968] [Q&A] [answered]
As far as I can see FT stores all its settings by default in directory

C:\Users\<username>\AppData\Roaming\FreeTube\

How can I tell FT to store it in a sub folder "mysettings" of FT installation directory?

If this is currently not possible: Can this option be added in the next release?

Thank you
Peter

**Comment by absidue** [comment, marked as answer]
Chromium has a `--user-data-dir` option that you can use to make it use a different location but you have to pass it every single time you run FreeTube, even when you use the URL scheme (you'll need to edit the registry to fix that), e.g. redirecting YouTube links from your browser.

Please note however that the FreeTube team does not support using any location other than the official ones, so if you lose your data or things don't work properly with a different location, you are on your own.

**Reply by absidue** [reply]
Also important to mention is that the installer replaces the installation directory, so if you place your data in there and run the installer again, it will get deleted. So if you want to do things like that only do so with the zip or 7z downloads, not with the installer, as then you are in control of what files get changed during an upgrade.

**Comment by pstein** [comment]
When I type

FreeTube.exe --user-data-dir=D:\tools\free tube downloader\mysettings

...and change some settings and exit the nothing is written to the new empty folder 

D:\tools\free tube downloader\mysettings

Whats wrong?

Do I have to use a trailing backslash?
